contracts/interfaces/IInterchainToken.sol

```diff
@@ -2,12 +2,20 @@
 
 pragma solidity ^0.8.0;
 
-import { IERC20 } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IERC20.sol';
+import { ITokenManager } from './ITokenManager.sol';
 
 /**
  * @dev Interface of the ERC20 standard as defined in the EIP.
  */
-interface IInterchainToken is IERC20 {
+interface IInterchainToken {
+
+    /**
+     * @notice Getter for the tokenManager used for this token.
+     * @dev Needs to be overwitten.
+     * @return tokenManager the TokenManager called to facilitate cross chain transfers.
+     */
+    function getTokenManager() external view returns (ITokenManager tokenManager);
```

Comment:
```suggestion
    function getTokenManager() external view returns (address tokenManager);
```
removes the interface dep (which itself has a lot of deps) so it's more self contained. a bit more incovenient in the main contract but seems to be an acceptable trade off